sqlmesh/core/model/decorator.py

```diff
@@ -82,10 +84,18 @@ def model(
             if statements:
                 common_kwargs[key] = [exp.maybe_parse(s, dialect=dialect) for s in statements]
 
+        if common_kwargs.get("depends_on"):
+            common_kwargs["depends_on"] = {
```

Comment:
why did you add this? if we weren't normalizing here before, it means this happens elsewhere

Reply:
+1. Seems like this should happen downstream (eg. in the model definition)